src/xrGame/ui/ChangeWeatherDialog.cpp

```diff
@@ -86,16 +86,26 @@ void ChangeWeatherDialog::InitChangeWeather(CUIXml& xmlDoc)
     auto& gameWeathers = gMapListHelper.GetGameWeathers();
     Initialize(gameWeathers.size());
     weathers.resize(gameWeathers.size());
+    
+    // There is no mistake. Vanilla algorithm sorted config keys by alphabet.
+    // We did it for vanilla bug fixes and compataible with game data.
```

Comment:
Can you please send this change and all related to it as a separate pull request?